libs/market-depth/src/lib/orderbook-manager.tsx

```diff
@@ -30,9 +31,10 @@ export const OrderbookManager = ({ marketId }: OrderbookManagerProps) => {
   const variables = { marketId };
   const resolutionRef = useRef(resolution);
   const [orderbookData, setOrderbookData] = useState<OrderbookData>({
-    rows: null,
+    asks: null,
+    bids: null,
   });
-  const dataRef = useRef<OrderbookData>({ rows: null });
+  const dataRef = useRef<OrderbookData>({ asks: null, bids: null });
```

Comment:
I realise this is what we have currently, but it seems odd to need to store the data both in useState and as a ref. Trying to understand why exactly we need to do this, but something seems wrong here

Reply:
refs we need for better memoization in hooks